docs/getting-started/faq.md

```diff
@@ -155,12 +156,12 @@ In other words your limit is likely the available memory in your system.
 
 If you use keys with limited time to live (Redis expires) this is normal behavior. This is what happens:
 
-* The master generates an RDB file on the first synchronization with the replica.
-* The RDB file will not include keys already expired in the master, but that are still in memory.
-* However these keys are still in the memory of the Redis master, even if logically expired. They'll not be considered as existing, but the memory will be reclaimed later, both incrementally and explicitly on access. However while these keys are not logical part of the dataset, they are advertised in `INFO` output and by the `DBSIZE` command.
-* When the replica reads the RDB file generated by the master, this set of keys will not be loaded.
+* The primary generates an RDB file on the first synchronization with the replica.
+* The RDB file will not include keys already expired in the primary but are still in memory.
+* These keys are still in the memory of the Redis primary, even if logically expired. They'll not be considered as existing  but the memory will be reclaimed later - both incrementally and explicitly on access. While these keys are not logically part of the dataset, they are advertised in `INFO` output and by the `DBSIZE` command.
+* When the replica reads the RDB file generated by the primary, this set of keys will not be loaded.
 
-As a result of this, it is common for users with many keys with an expire set to see less keys in the replicas, because of this artifact, but there is no actual logical difference in the instances content.
+As a result of this, it is common for users with many keys with an expire set to see less keys in the replicas. Because of this artifact, there is no actual logical difference in the instances content.
```

Comment:
```suggestion
As a result of this, it is common for users with many keys with an expire set to see fewer keys in the replicas. Because of this artifact, there is no actual logical difference in the instance's content.
```